manifests/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -4,7 +4,7 @@ kind: ClusterRole
 metadata:
   name: istio-pilot-{{ .Release.Namespace }}
   labels:
-    app: pilot
+    app: istiod
```

Comment:
Do we want to continue keep the istio-pilot-* resources labeled with pilot? I think there should be istiod equivalents for most (all?) of them. It will make clean-up a bit easier since we can prune based on app=pilot.

Reply:
Good point, I just did a sed over everything. Those should probably have the old label. Ill send a follow up